holistic_bias/INSTRUCTIONS-count-biases.md

```diff
@@ -0,0 +1,52 @@
+# Multilingual Holistic Biases Counts
+
+## Setup 
```

Comment:
Would be useful to add a paragraph or two to the top of this explaining what this functionality is and how it works